packages/nitro-protocol/gas-benchmarks/vanillaSetup.ts

```diff
@@ -49,3 +60,25 @@ beforeEach(async () => {
 afterAll(async () => {
   await ganacheServer.close();
 });
+
+expect.extend({
+  async toConsumeGas(
+    received: any, // TransactionResponse
+    benchmark: number
+  ) {
+    const {gasUsed} = await received.wait();
+    const pass = (gasUsed as BigNumber).eq(benchmark); // This could get replaced with a looser check with upper/lower bounds
+    if (pass) {
+      return {
+        message: () => `expected to NOT consume ${benchmark} gas, but did`,
```

Comment:
```suggestion
        message: () => `expected to consume ${benchmark} gas, and did`,
```
I've read this about 4 times and one of us has something backwards in our head.

Reply:
Yeah it's a bit confusing but this is the slot where you put the message to be printed in the case `expect().not.toConsumeGas()` fails. 

See this article https://everyday.codes/javascript/create-custom-jest-matchers-to-test-like-a-pro/ .

Reply:
(No message is printed when `pass==true`)